test/strategy/anchor/fork/AnchorStrategy.fork.test.ts

```diff
@@ -132,7 +131,6 @@ describe('AnchorStrategy Mainnet fork', () => {
         depositOperations.operator,
         expectAUstReceive,
       );
-      await strategy.finishDepositStable(0);
```

Comment:
I'm confused, why are you taking these out? The idea was to keep the tests as they were, to prove that the strategy should still work with our current Vault

Reply:
Because we wont have a finishDepositStable method for the yearn strategy right?

Reply:
Because we wont have a finishDepositStable method for the yearn strategy right?